js/faq.js

```diff
@@ -1,8 +1,34 @@
 const faqLinks = document.querySelectorAll('.faq_link');
 const currentLocation = window.location.hash;
 
+window.addEventListener('keydown', (e) => {
+  const ancTag = document.querySelector(
+    `a[href="${e?.target?.attributes?.href?.value}"]`,
+  );
+  if (e.key === ' ' && ancTag) {
+    e.preventDefault();
+    ancTag?.click();
+  }
+});
+
+const removeForOthers = (target) => {
```

Comment:
Could this be renamed to `removeFocusForOthers` or something more appropriate name?

Reply:
changed